app/helpers/people_helper.rb

```diff
@@ -15,6 +15,14 @@ def format_person_email(person)
     mail_to(person.email)
   end
 
+  def format_person_login_status(person)
+    status = person.login_status
+    icon(
+        I18n.t("people.login_status.#{status}.icon"),
```

Comment:
Gehört der icon key wirklich in die translations? Ich hätt das jetzt eher in einen Hash mit `{ status: icon-key }` gepackt.

Reply:
Ja, guter Punkt, man will ja wohl kaum pro Sprache ein anderes Icon verwenden.